.contributing/code-of-conduct.md

```diff
@@ -4,10 +4,10 @@
 
 #### Reviews 
 
-- 1 review is required for merging 
+- Generally only 1 approval is required before merging.  The exceptions are: 
+  - Files in the `.contributing/` folder, where approvals are needed from the whole team before merging
```

Comment:
throw('numReviewsNotMetException'); 😄

Reply:
Haha we need to get you out of the error handling area for an evening or two mate 😄